### PR TITLE
(maint) fix ssl cleanup

### PR DESCRIPTION
**Pull request by ciprianbadescu**
asterisk will not be expanded inside single thicks

### CODE REVIEW

File: setup/common/040_ValidateSignCert.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
   hosts.each do |host|
     ssldir = on(host, puppet('agent --configprint ssldir')).stdout.chomp
     # preserve permissions for master's ssldir so puppetserver can read it
-    on(host, "rm -rf '#{ssldir}/*'")
+    on(host, "rm -rf '#{ssldir}/'*")
```
Comment (ciprianbadescu) [inline setup/common/040_ValidateSignCert.rb]: did some tests on Windows/Linux with directory containing spaces, we need to keep `#{ssldir}` in single thicks

**Comment by mchllweeks** [comment]
@puppetlabs-jenkins retest this please

**Comment by kevpl** [comment]
think this might have been tested before #124 was finished, rekicking to double-check.

@puppetlabs-jenkins retest this please

**Comment by ciprianbadescu** [comment]
retest this please

**Comment by ciprianbadescu** [comment]
retest this please

**Pull request merged**